index.bs

```diff
@@ -5771,6 +5990,17 @@ To <dfn>process a network event</dfn> given |session|, |event|, and |request|:
 
 </div>
 
+#### The network.BodyType Type #### {#type-network-BodyType}
+
+[=Remote end definition=] and [=local end definition=]
+
+<pre class="cddl remote-cddl local-cddl">
+network.BodyType = "request" / "response"
```

Comment:
This spec version does not support the request body type yet?

Reply:
Correct, we could remove "request" for now